packages/react-preferences/src/components/BusinessFooter.tsx

```diff
@@ -31,46 +31,49 @@ const FooterWrapper = styled.div`
   }
 `;
 
-const Button = styled.a`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  white-space: nowrap;
-  gap: 2.5px;
-  padding: 5px 50px;
-  border: 0.5px solid #ffffff;
-  border-radius: 5px;
-  :hover {
-    cursor: pointer;
-  }
-  p {
-    margin: 0;
-    font-size: 12px;
-    font-weight: 600;
-  }
-`;
-
 const SVGWrapper = styled.div`
   margin: 20px;
   display: flex;
   gap: 12.5px;
   justify-content: center;
   align-items: center;
 
-  > svg {
+  svg {
     width: 20px;
   }
 `;
 
-export const BusinessFooter: React.FunctionComponent = () => {
+export const BusinessFooter: React.FunctionComponent<{ links: any }> = ({
+  links,
+}) => {
   return (
     <FooterWrapper>
       <SVGWrapper>
-        <Facbebook />
-        <Instagram />
-        <LinkedIn />
-        <Medium />
-        <Twitter />
+        {links?.facebook?.url && (
```

Comment:
This feels a bit awkward datastrucuture-wise. might be easier to map this to a  strongly typed collection on the API side.

```ts
type SocialLinks = Array<{type: "facebook" | "Instagram" | string, value: string}>
```